src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -272,16 +264,40 @@ const InstancePage = (): JSX.Element => {
                   <TabTitleText>{t("common.errorHandling")}</TabTitleText>
                 )
               }
-            >
-              <PageSection>
-                <ErrorHandlingTabContent
-                  bridge={bridge}
-                  isBridgeLoading={isBridgeLoading}
-                />
-              </PageSection>
-            </Tab>
+            />
           </Tabs>
         </PageSection>
+        <PageSection isFilled padding={{ default: "noPadding" }}>
+          <TabContent
+            key={INSTANCE_PAGE_TAB_KEYS.processors}
+            eventKey={INSTANCE_PAGE_TAB_KEYS.processors}
+            id={"instance-page__tabs-processors"}
+            ouiaId="processors"
+            activeKey={activeTabKey}
+          >
+            {activeTabKey === INSTANCE_PAGE_TAB_KEYS.processors && (
+              <ProcessorsTabContent
+                instanceId={instanceId}
+                pageTitle={getPageTitle(bridge)}
+              />
+            )}
+          </TabContent>
+          <TabContent
+            key={INSTANCE_PAGE_TAB_KEYS["error-handling"]}
+            eventKey={INSTANCE_PAGE_TAB_KEYS["error-handling"]}
+            id={"instance-page__tabs-error-handling"}
+            ouiaId="error-handling"
+            activeKey={activeTabKey}
+            className={"pf-c-page__main-section pf-u-h-100"}
```

Comment:
Altering `TabContent` styles to make its content fill the available page height. 

Adding the `pf-c-page__main-section` to make it behave like a `PageSection`'s, and `pf-u-h-100`, a PF css utility class to set the height to `100%`